internal/hcsoci/resources_lcow.go

```diff
@@ -18,6 +18,7 @@ import (
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 )
 
+const mountDir = "mounts"
```

Comment:
Lets make this a:

```go
const (
    lcowMountsPrefix = "/mounts/m/%d"
)
```

Then we can use: `path.Join(rootInUvm, fmt.Sprintf(lcowMountsPrefix, strconv.Iota(i))`